src/v2/interpolation.rs

```diff
@@ -101,7 +101,8 @@ fn interpolate_helper(input: &str, mode: Mode, env: &Environment) -> Result<Stri
             "".to_owned()
         } else {
             // Handle actual interpolations.
-            let var = caps.name("var1")
+            let var = caps
+                .name("var1")
```

Comment:
Normally I would recommend running `cargo fmt` before submitting a patch, but this patch mixes up three things:

1. Running the code through `cargo fmt` for the first time (or at least the first time in a very long time).
2. Updating to Rust 2018 (and re-running `cargo fmt` as needed afterwards).
3. Updating dependencies.

Ideally, this could be done with one PR that contains exactly three patches: one addressing each issue above. And each patch would show all tests passing.

It's OK if you don't have time to do things this way! But that's how I want to update `compose_yml`, so that I can keep those issues separate and review each part of the change accordingly.

Anyway, thank you for your patch, and for figuring out all of the things needed to get the tests passing again. That's a lot of hard work and I really appreciate it!

Reply:
I can revert 'cargo fmt' changes for this PR if you prefer to do it separately. Let me know...